test/ViewController.swift

```diff
@@ -0,0 +1,178 @@
+//
+//  ViewController.swift
+//  test
+//
+//  Created by apple on 05/06/2018.
+//  Copyright © 2018 wolfpack. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController, DidFinishAnimating
+{
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        game.cardGenerator()
```

Comment:
create a multiple methods for all commands in the viewDidLoad and call them in viewDidLoad